app/views/partials/trials-list.html

```diff
@@ -93,7 +95,7 @@
             {% endif %}
           {% endif %}
           {% if trial.isInProgress %}
-          <p>{{ trial.daysAfterStart }} days <span>until completed</span></p>
+          <p>{{ trial.daysBeforeCompletion }} days <span>until completed</span></p>
```

Comment:
It's merely a cosmetic change, so it would be similar to `daysAfterCompletion` and easier to follow. Lots of trials' variables in this code. (e.g. it's the same thing, just with a different hat)